models/articles.models.js

```diff
@@ -1,28 +1,43 @@
 const db = require("../db/connection");
 
 fetchArticle = (article_id) => {
-  return db
-    .query(
-      `SELECT articles.*, COUNT(comments.comment_id)::int AS comment_count
+  if (isNaN(Number(article_id))) {
+    return Promise.reject({
+      status: 404,
+      msg: "Bad path. Article ID should be a number",
+    });
+  } else {
```

Comment:
technically can do without an else here as we're returning if the if block is entered :)